src/init/features/frameworkstack/constants.ts

```diff
@@ -0,0 +1,15 @@
+export const DEFAULT_REGION = "us-central1";
+export const ALLOWED_REGIONS = [
+  { name: "us-east1 (South California)", value: "us-east1" },
+  { name: "us-east4 (Northern Virginia)", value: "us-east4" },
+  { name: "us-east5 (Columbus)", value: "us-east5" },
+  { name: "us-central1 (Iowa)", value: "us-central1" },
+  { name: "us-central2 (Chicago, IL)", value: "us-central2" },
+  { name: "us-south1 (Dallas)", value: "us-south1" },
+  { name: "us-west1 (Oregon)", value: "us-west1" },
```

Comment:
Where did we get this list?

If we are making this up, then I'd just pick `us-central1` only

Reply:
This was according to doc shared here: https://www.figma.com/file/soCklCIUh3XVQQZPQm9Olx/Firestack?type=design&node-id=1004-6227&t=q4xvLZwgtAsR9hbn-0